### DISCUSSION TITLE
Delay doesn't work

### DISCUSSION THREAD

**Discussion by johnsonthedev** [clauderic/dnd-kit#82] [Q&A] [answered]
Hi I tried to add a delay to my sensors but it's not working. Anyone knows why ?

Here is the sandbox: https://codesandbox.io/s/fervent-feather-3twfd?file=/src/Sortable.js:1122-1589

```js

  const sensors = useSensors(
    useSensor(MouseSensor, {
      activationConstraint: {
        delay: 250
      }
    }),
    useSensor(PointerSensor, {
      activationConstraint: {
        delay: 250
      }
    }),
    useSensor(KeyboardSensor, {
      coordinateGetter: sortableKeyboardCoordinates,
      activationConstraint: {
        delay: 250
      }
    }),
    useSensor(TouchSensor, {
      activationConstraint: {
        delay: 250
      }
    })
  );

```

**Comment by clauderic** [comment, marked as answer]
You should not be using both `PointerSensor` and `TouchSensor` or `MouseSensor` at the same time. Valid combinations are:

```
- PointerSensor
- KeyboardSensor
```
or
```
- MouseSensor
- TouchSensor
- KeyboardSensor
```

In either case, the delay seems to work fine for me, try increasing the value of the `delay` to validate https://codesandbox.io/s/serene-zhukovsky-qiu5x?file=/src/Sortable.js

**Reply by bayhalldigital** [reply]
This is really useful to know, I didn't see this in the documentation and fixed my desktop/mobile app drag and drop issues.
